### ISSUE TITLE
(Linux): Install JSON files on the right place for Chrome (and add Chromium support)

### ISSUE THREAD

**Issue by klondi** [web-eid/web-eid-app#308]
Currently, the native-messaging json file for chrome is installed on /usr/share/web-eid as a result after installing the extension it will not work as it cannot run.

Instead as stated on https://developer.chrome.com/docs/extensions/develop/concepts/native-messaging the JSON file should be installed on /etc/opt/chrome/native-messaging-hosts

With this adding Chromium support is fairly easy too. According to https://developer.chrome.com/docs/extensions/how-to/distribute/install-extensions ncibgoaomkmdpilpocfeponihegamlic.json should be installed on /usr/share/chromium/extensions/ and to allow native messaging https://developer.chrome.com/docs/extensions/develop/concepts/native-messaging requires installing eu.webeid.json into /etc/chromium/native-messaging-hosts/

In summary to address these issues:
Install eu.webeid.json into both /etc/opt/chrome/native-messaging-hosts and /etc/chromium/native-messaging-hosts/
Install ncibgoaomkmdpilpocfeponihegamlic.json also into /usr/share/chromium/extensions/

**Comment by mrts** [comment]
Sorry for the delayed response! We will discuss this internally and get back to you.

**Comment by sergey-abc** [comment]
Spent lot of time trying to understand why web-eid is not executed from Chromium. 
As a result I see the message that it's not possible to use outdated ID card software. 
Thanks for solution @klondi !

**Comment by mrts** [comment]
Thanks for your contribution! We intend to release Chromium support in the upcoming 2.6 release.
